docs/getting-started.rst

```diff
@@ -79,6 +87,14 @@ displayed using this command::
                           Set amount of displayed information
     -s, --stop-on-failure
                           Stop running checks after a failed one.
+    --map-charm MAP_CHARM
+                          WARNING: This option can lead to failed verifications
+                          when used incorrectly. This option allows user to
+                          explicitly specify which charm is the application
```

Comment:
Suggestion: "This option allows users to explicitly specify the charm used by an application"